app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    before_action :configure_sign_up_params, only: [:create] # rubocop:disable Rails/LexicallyScopedActionFilter
+    before_action :configure_account_update_params, only: [:update]
+
+    # GET /resource/sign_up
+    # def new
+    #   super
+    # end
+
+    # POST /resource
+    # def create
+    #   super
+    # end
+
+    # GET /resource/edit
+    # def edit
+    #   super
+    # end
+
+    # PUT /resource
+    def update
+      super do |resource|
+        resource.avatar.purge if params[:user][:avatar].nil? && params[:user][:avatar_present] == 'false'
```

Comment:
This piece was taken from the old `UsersController` to delete avatar images.

Reply:
Fine for me, thanks for pointing that out